### DISCUSSION TITLE
Support Background Gradient during `selection:`

### DISCUSSION THREAD

**Discussion by tiarebalbi** [tailwindlabs/tailwindcss#12601] [Ideas]
Would be great if we could support background gradient during `selection:` usage.

Example:
https://play.tailwindcss.com/sRuNtWkqol

**Comment by wongjn** [comment]
`selection:` corresponds to the `::selection` pseudo-element, but as of time of writing, [according to MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/::selection#allowable_properties):

> Only certain CSS properties can be used with `::selection`:
>
> - [`color`](https://developer.mozilla.org/en-US/docs/Web/CSS/color)
> - [`background-color`](https://developer.mozilla.org/en-US/docs/Web/CSS/background-color)
> - [`text-decoration`](https://developer.mozilla.org/en-US/docs/Web/CSS/text-decoration) and its associated properties
> - [`text-shadow`](https://developer.mozilla.org/en-US/docs/Web/CSS/text-shadow)
> - [`-webkit-text-stroke-color`](https://developer.mozilla.org/en-US/docs/Web/CSS/-webkit-text-stroke-color), [`-webkit-text-fill-color`](https://developer.mozilla.org/en-US/docs/Web/CSS/-webkit-text-fill-color) and [`-webkit-text-stroke-width`](https://developer.mozilla.org/en-US/docs/Web/CSS/-webkit-text-stroke-width)
>
> In particular, [`background-image`](https://developer.mozilla.org/en-US/docs/Web/CSS/background-image) is ignored.

Thus, `background-image` would need to be supported on `::selection`. If it ever does so, it should work automatically with Tailwind.
